package.json

```diff
@@ -47,7 +47,7 @@
     "node": ">=16.5.0"
   },
   "devDependencies": {
-    "@babel/types": "^7.12.0",
+    "@babel/types": "^7.17.0",
```

Comment:
I don't think the dependency change is needed in this PR. Please remove this and the lockfile changes.

Reply:
It must be upgraded. Otherwise `ImportSpecifier` doesn't have the `importKind` property

Reply:
You are right, the original version did not cause the error, I will remove this change